GitUI/Theming/Renderers/ButtonRenderer.cs

```diff
@@ -280,21 +280,6 @@ public enum CheckBox
                 CBS_EXCLUDEDPRESSED = 19,
                 CBS_EXCLUDEDDISABLED = 20,
             }
-
```

Comment:
Theme related, really remove?

Reply:
As an option, if we want to keep these, we can move all these unused theme-related types under `#if SUPPORT_THEME_HOOKS`.

Reply:
IMO u should remove them. then mark that commit with a tag. if u ever want to bring them back u can find them via the tag, and then revert the commit

Reply:
We've hidden all theme-related stuff behind `SUPPORT_THEME_HOOKS` pragma. Having all of those under the same pragma will make it easier to bulk-remove or bulk-reenable in the future.

Reply:
I suggest this PR is changed to only include the first 5 removed files.

The theming related should be adding SUPPORT_THEME_HOOKS.
I still hope theming can be reenabled, then it is easier if the code is not removed.
However if removed, remove all in one dedicated commit, not many. There are probably other theming related methods that will be removed over time.